packages/leap/lib/src/input.dart

```diff
@@ -41,13 +41,22 @@ class SimpleCombinedInput extends Component
       _appFocused &&
       (_tapInput.isPressedRight || _keyboardInput.isPressedRight);
 
-  SimpleCombinedInput() {
+  SimpleCombinedInput({
+    Set<PhysicalKeyboardKey>? leftKeys,
+    Set<PhysicalKeyboardKey>? rightKeys,
+  }) {
```

Comment:
I think it would be better to not leak the details of the `SimpleKeyboardInput` params into this parent enclosing class. Can you instead just take `this._keyboardInput = SimpleKeyboardInput()` as a param here?

Reply:
Good idea! Changed!